lambda-buffers-proto/compiler.proto

```diff
@@ -539,49 +539,56 @@ message KindCheckError {
     VarName var_name = 2;
   }
 
-  // Unification has failed - type is incorrectly defined. Error reads as 
+  // Unification has failed - type is incorrectly defined. Error reads as
   // follows:
-  // > In ty_name definition an error has occurred when trying to unify kind 
-  // > ty_kind_1 with ty_kind_2. 
+  // > In ty_name definition an error has occurred when trying to unify kind
+  // > ty_kind_1 with ty_kind_2.
   //
-  // FIXME(cstml): add source of constraint to the error such that user can see 
-  // where the constraint was generated - therefore where the error precisely 
-  // is. 
+  // FIXME(cstml): add source of constraint to the error such that user can see
+  // where the constraint was generated - therefore where the error precisely
+  // is.
   message ImpossibleUnificationError {
     TyName ty_name = 1;
     Kind ty_kind_1 = 2;
     Kind ty_kind_2 = 3;
   }
 
   // Error reads:
-  // Inifinitely recursive term detected in definition ty_name. 
+  // Inifinitely recursive term detected in definition ty_name.
   message RecursiveKindError {
     TyName ty_name = 1;
   }
-  
+
   // The inferred type differs from the type as defined.
   message InconsistentTypeError {
     TyName ty_name = 1;
     Kind inferred_kind = 2;
     Kind defined_kind = 3;
   }
 
+  // A tydef was defined multiple times.
```

Comment:
Not really, `Multiple TyDefs with the same TyName were found`

Try to adopt the documentation pattern that we use elsewhere. For error messages, lose the ad hoc in-doc notation and just put what the error reads as:

```
// Multiple TyDefs with the same TyName were found in ModuleName.
message MultipleTyDefError {
  // Module in which the error was found.
  ModuleName module_name = 1;
  // Conflicting type definitions.
  repeated TyDef ty_defs = 2;
}
```

Also use a consistent vocabulary, not declaration, but TyDef (definition).